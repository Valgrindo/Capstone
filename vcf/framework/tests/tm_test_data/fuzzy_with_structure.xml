<!-- Test that no structure is allowed to be supplied to a fuzzy component -->
<commands>

<command name="FUZZY_STRUCT">
    <component indicator="SPEECHACT" type="SA_REQUEST">
        <role name="CONTENT">

            <component indicator="F" word="SHOW" map_param="verb">
                <role name="AGENT1">
                    <component type="PERSON" map_param="agent"/>
                </role>
                <role name="NEUTRAL">

                    <component type="FUZZY_COMP" fuzzy="True">
                        <role name="ILLEGAL">
                            <component/>
                        </role>
                    </component>
                </role>
            </component>

        </role>
    </component>
</command>


</commands>