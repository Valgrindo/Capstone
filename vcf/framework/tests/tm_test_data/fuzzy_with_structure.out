Cannot assign structure to fuzzy component <component * FUZZY_COMP>